Woof/WoofTests/Model/Preferences/RoleTests.swift

```diff
@@ -0,0 +1,8 @@
+import XCTest
+
+final class RoleTests: XCTestCase {
+    func testAllRolesExistInAPI() {
+        _ = Role.owner
+        _ = Role.sitter
```

Comment:
Should we add the test case for `none`?
Should we use the [CaseIterable](https://developer.apple.com/documentation/swift/caseiterable) protocol and its `allCases` property to check the existence of all roles?

Reply:
Added the conformation to CaseIterable.